demo/app/ios/Runner/OpenID4CI.swift

```diff
@@ -12,13 +12,15 @@ public class OpenID4CI {
     private var didResolver: ApiDIDResolverProtocol
     private var crypto: ApiCryptoProtocol
     private var activityLogger: ApiActivityLoggerProtocol
+    private var kms: LocalkmsKMS
```

Comment:
`LocalkmsKMS` is a rough naming...is there anything we can do it avoid repeating ourselves?

Reply:
Unfortunately no, this is how ios binding is generated @DRK3 have tried the options but this is how the naming is for ios binding

Reply:
Yes indeed, this one was tricky. For iOS bindings, gomobile uses the Go package name as a suffix for all symbols from that package. The package name is `localkms` and the type exposed from that package is `KMS`, hence why we get `LocalkmsKMS` (gomobile capitalizes it automatically). The tricky thing is choosing package+type names that look reasonable for iOS + Android + the Go source code. I managed to improve some of the other packages with careful name choices, but at the time I didn't have a great solution for this one. Any ideas @birtony?